docs/bokeh/source/docs/user_guide/interaction/js_callbacks.rst

```diff
@@ -82,28 +82,120 @@ JavaScript code to run in the browser when an event occurs.
 
     from bokeh.models.callbacks import CustomJS
 
-    callback = CustomJS(args=dict(xr=plot.x_range), code="""
+    callback = CustomJS(args=dict(xr=plot.x_range, yr=plot.y_range, slider=slider), code="""
+    // imports
+    import {some_function, SOME_VALUE} from "https://cdn.jsdelivr.net/npm/package@version/file"
 
-    // JavaScript code goes here
+    // constants, definitions and state
+    const MY_VALUE = 3.14
 
-    const a = 10;
+    function my_function(value) {
+        return MY_VALUE*value
+    }
 
-    // the model that triggered the callback is cb_obj:
-    const b = cb_obj.value;
+    class MyClass {
+        constructor(value) {
+            this.value = value
+        }
+    }
+
+    let count = 0
+
+    // the callback function
+    export default (args, obj, data, context) => {
+        count += 1
+        console.log(`CustomJS was called ${count} times`)
 
-    // models passed as args are automagically available
-    xr.start = a;
-    xr.end = b;
+        const a = args.slider.value
+        const b = obj.value
 
+        const {xr, yr} = args
+        xr.start = my_function(a)
+        xr.end = b
+    }
     """)
 
-Note that in addition to the ``code`` property, ``CustomJS`` also accepts
-an ``args`` property that maps string names to Bokeh models. Any Bokeh
-models that are configured in ``args`` (on the "Python side") will
-automatically be available to the JavaScript code by the corresponding name.
+The code snippet must contain a default export, which must be a function defined
+either using arrow function syntax ``() => {}`` or a classical function syntax
+ ``function() {}``. Depending on the context, this function may be an async
```

Comment:
```suggestion
``function() {}``. Depending on the context, this function may be an async
```

The one space makes Sphinx think you want to quote something. Sphinx warns you with:

```
~/bokeh/docs/bokeh/source/docs/user_guide/interaction/js_callbacks.rst:120: ERROR: Unexpected indentation.
~/bokeh/docs/bokeh/source/docs/user_guide/interaction/js_callbacks.rst:121: WARNING: Block quote ends without a blank line; unexpected unindent.
```

which should be silenced by this change.

And this is how it looks right now

![grafik](https://github.com/bokeh/bokeh/assets/68053396/84dc0236-e55c-4a3a-98f6-fc04a739d6fd)